cmake/config.cmake

```diff
@@ -151,6 +151,9 @@ set(USE_CUDNN OFF)
 # Whether use cuBLAS
 set(USE_CUBLAS OFF)
 
+# Whether use cuSPARSE
+set(USE_CUSPARSE OFF)
```

Comment:
Just curious. If cuSPARSE comes from the normal CUDA, can we simplify this tag by always enabling it when USE_CUDA is ON? I have the same concern as cuBLAS too several months ago but didn't get a chance to follow up.

Reply:
btw, based on this flag the CI still needs to be updated, because `config.cmake` on the CI is fixed.

Reply:
Thanks for the reminding! will fix it!
and I think by specifying the USE_CUBLAS and USE_CUSPARSE flags, it can still help TVM do not link some unneeded lib files.
So I guess the those two flags still fit the purpose?